recipes/future/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "0.15.2" %}
+
+package:
+  name: future
+  version: {{ version }}
+
+source:
+  fn: future-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/f/future/future-{{ version }}.tar.gz
+  sha256: 3d3b193f20ca62ba7d8782589922878820d0a023b885882deec830adbf639b97
+
+build:
+  number: 1
+  entry_points:
+    - futurize = libfuturize.main:main
+    - pasteurize = libpasteurize.main:main
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  commands:
+    - futurize -h
+    - pasteurize -h
+  imports:
+    - future
+
+about:
+  home: http://python-future.org
+  license: MIT
+  license_file: LICENSE.txt
+  summary: Clean single-source support for Python 3 and 2
+  description: |
+    Python-future  is the missing compatibility layer between Python 2 and
+    Python 3. It allows you to use a single, clean Python 3.x-compatible
+    codebase to support both Python 2 and Python 3 with minimal overhead.
+  doc_url: http://python-future.org/
+  dev_url: https://github.com/PythonCharmers/python-future
```

Comment:
You need to add the `extra/maintainer` section here. See https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L41-L45 for an example.